Exercises/9-array.js

```diff
@@ -1,7 +1,10 @@
 'use strict';
 
-const phonebook = null;
+const phonebook = [{name: 'Timur Garashenko', phone: '+380445554433'}, {name: 'Artem Trifomovisch', phone: '+380636664499'}];
```

Comment:
Split to multiline

Reply:
I fix all! And after fixing. I pull requsted all. I used new rules npm, hpw. Examples on commit down.